oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -416,6 +416,12 @@ Apache-ShenYu:
 Hutool:
   id: 128
   languages: Java
+https:
+  id: 129
+  languages: Java,C#,Node.js,Python
+TLS:
+  id: 130
+  languages: Java,C#,Node.js,Python
```

Comment:
In the mesh protocol, TLS has two modes, do we need to have them too at here?

https://github.com/apache/skywalking-data-collect-protocol/blob/906a834dc0c3fb66719b8313f8f9298be3a27349/service-mesh-probe/service-mesh.proto#L54-L55

`http=49` is in lower case, `https=129` is in lower case, why `tls` is upper case?

I think you don't need to list these languages here, right? You should list `ebpf`?

Reply:
> In the mesh protocol, TLS has two modes, do we need to have them too at here?

For now, we only could support analyzing the TLS. mTLS is not supported.

Reply:
> `http=49` is in lower case, `https=129` is in lower case, why `tls` is upper case?
> 
> I think you don't need to list these languages here, right? You should list `ebpf`?

Updated.